pkg/client/handlers.go

```diff
@@ -25,7 +26,8 @@ func (c *Client) internalHandlers() {
 	c.Config.HandleAPIpath("", "version", c.versionResponse, "GET", "HEAD")
 	c.Config.HandleAPIpath("", "info", c.updateInfo, "PUT")
 	c.Config.HandleAPIpath("", "info/alert", c.updateInfoAlert, "PUT")
-	c.Config.HandleAPIpath("", "cfsync", c.handleCFSyncReq, "GET")
+	c.Config.HandleAPIpath("", "cfsync", c.handleCFSyncReq, "GET") // deprecated
```

Comment:
We'll remove this one later, keep using it for now if you think people push that button a lot.

Reply:
You can remove it, i did.